wallaby/app/assets/stylesheets/wallaby/base.scss

```diff
@@ -1,8 +1,9 @@
 
 // NOTE: "bootstrap-sprockets" must be imported
 // before "bootstrap" and "bootstrap/variables"
-@import 'core';
 @import 'base/vars';
```

Comment:
Move `base/vars` to the beginning so that the vars needed by `core` are declared